src/BugsnagUnity/IDelivery.cs

```diff
@@ -61,7 +61,7 @@ IEnumerator PushToServer(IPayload payload, byte[] body)
     {
       using (var req = new UnityWebRequest(payload.Endpoint.ToString()))
       {
-        req.SetRequestHeader("Content-Type", "application/json; charset=utf-8");
+        req.SetRequestHeader("Content-Type", "application/json");
```

Comment:
do we not need to set the charset anymore? If this fails a mazerunner step, we should update the mazerunner assertion

Reply:
It is indeed due to maze-runner, I don't think it's necessary though as JSON is by default (possibly only?) in utf8